lte/gateway/c/core/oai/tasks/ngap/ngap_amf.c

```diff
@@ -24,7 +24,7 @@
 #include "lte/gateway/c/core/oai/common/itti_free_defined_msg.h"
 #include "lte/gateway/c/core/oai/common/log.h"
 #include "lte/gateway/c/core/oai/include/amf_config.hpp"
-#include "lte/gateway/c/core/oai/include/mme_config.h"
+#include "lte/gateway/c/core/oai/include/mme_config.hpp"
```

Comment:
can you move
https://github.com/pruthvihebbani/magma/blob/cf4c51e84a4255b83635b3ce8046ed439875f486/lte/gateway/c/core/oai/tasks/ngap/ngap_amf.c#L48
to the top